modules/pre-commit.nix

```diff
@@ -117,12 +117,18 @@ let
                 description = "Whether to pass filenames as arguments to the entry point.";
                 default = true;
               };
+            stages =
+              mkOption {
+                type = types.listOf types.str;
+                description = "Confines the hook to run at a particular stage.";
+                default = cfg.default_stages;
```

Comment:
```suggestion
                defaultText = (lib.literalExpression or lib.literalExample) "default_stages";
```

Reply:
Thanks, I've updated that now to match

Reply:
Removing the value for `default` broke the build as stages was left without a value, but I've added the `defaultText` in addition. Was that what you were intending?

Reply:
It is; I just used the suggestion feature wrong. Sorry about that.

Reply:
No worries, thanks for your help 😄